### PR TITLE
[UI] Move the interop tab to headers

### PR DESCRIPTION
**Pull request by KyleJu**
Fix https://github.com/web-platform-tests/wpt.fyi/issues/2287

### CODE REVIEW

**Comment by KyleJu** [comment]
@stephenmcgruer - I also experimented with lazy loading the interop view but encountered WebDriver tests timeout issues. With lazy loading, I saw a slight speedup on the loading (0.5~ second)

File: webapp/components/wpt-header.js [hunk 1]
```diff
@@ -75,6 +75,7 @@ class WPTHeader extends WPTFlags(PolymerElement) {
       <nav>
         <!-- TODO: handle onclick with wpt-results.navigate if available -->
         <a href="/">Latest Run</a>
+        <a href="/interop[[path]]?[[query]]">Interoperability</a>
```
Comment (stephenmcgruer) [inline webapp/components/wpt-header.js:78]: Let's move this to between 'Insights' and 'Processor'.

(Ultimately I think we'll want to shuffle things around, but for now this seems a bit more coherent than Latest / Interop / Recent / Insights / Processor. Plus we know people rarely use this view anyway.)

**Review by stephenmcgruer — APPROVED** [review]
This is awesome! It looks much cleaner, and it sort of gives me the idea that we could one day use the 'interoperability' page for things like BSFs/etc... (and then make it the default page, and then make 'latest results' not really a thing, and... ahem, getting ahead of myself)

**Comment by KyleJu** [comment]
`suites_medium_test.go` encountered a timeout erro `timed out starting Datastore emulator`. Retrying checks

**Pull request merged**